package.json

```diff
@@ -152,6 +152,7 @@
     "eslint-plugin-no-only-tests": "2.6.0",
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.4.0",
+    "fs-extra": "^10.0.1",
```

Comment:
I don't think this is supposed to be a dev dependency? Right? And can we depend on the exact version instead of the caret range?